plugin/trino-ranger/src/test/resources/ranger-servicedef-trino.json

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
As I understand it is part of Ranger 2.5.0 right? Can this file change in Ranger 3.0.0? How can we notice that if we update the ranger dependency in Trino?

Reply:
Yes, this service-def is part of Ranger 2.5.0.

As Trino authorization APIs evolve, the service-def in Ranger will have to be revised for potential update - for example when new resources are to be authorized, new permissions are added.